acceptance/bundle/migrate/var_arg/output.txt

```diff
@@ -0,0 +1,66 @@
+
+>>> DATABRICKS_BUNDLE_ENGINE=terraform [CLI] bundle deploy --var=job_name=Custom Job Name
+Uploading bundle files to /Workspace/Users/[USERNAME]/.bundle/migrate-var-test/dev/files...
+Deploying resources...
+Updating deployment state...
+Deployment complete!
+
+>>> [CLI] bundle deployment migrate --var=job_name=Custom Job Name
+Note: Migration should be done after a full deploy. Running plan now to verify that deployment was done:
+Plan: 0 to add, 0 to change, 0 to delete, 1 unchanged
+Success! Migrated 1 resources to direct engine state file: [TEST_TMP_DIR]/.databricks/bundle/dev/resources.json
+
+Validate the migration by running "databricks bundle plan --var 'job_name=Custom Job Name'", there should be no actions planned.
+
+The state file is not synchronized to the workspace yet. To do that and finalize the migration, run "bundle deploy --var 'job_name=Custom Job Name'".
+
+To undo the migration, remove [TEST_TMP_DIR]/.databricks/bundle/dev/resources.json and rename [TEST_TMP_DIR]/.databricks/bundle/dev/terraform/terraform.tfstate.backup to [TEST_TMP_DIR]/.databricks/bundle/dev/terraform/terraform.tfstate
```

Comment:
unrelated, would be nice we could print the bash commands directly here:
```
rm [TEST_TMP_DIR]/.databricks/bundle/dev/resources.json
mv [TEST_TMP_DIR]/.databricks/bundle/dev/terraform/terraform.tfstate.backup [TEST_TMP_DIR]/.databricks/bundle/dev/terraform/terraform.tfstate
```

As I user, I would be nervous otherwise mucking with state files.